### PR TITLE
Added JTD15 and JTD20 series of DC-DC Converters from XP Power

### PR DESCRIPTION
**Pull request by jamesrlucas**
Addition of footprint for JTD series of DC-DC Converters from XP Power. This includes JTD15 & JTD20 series, both single and dual output types

Datasheet links: [JTD20](https://www.xppower.com/portals/0/pdfs/SF_JTD20.pdf) & [JTD15](https://www.xppower.com/portals/0/pdfs/SF_JTD15.pdf)

image

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

File: Converter_DCDC.dcm [hunk 1]
```diff
@@ -2166,6 +2166,168 @@ K XP_POWER DC/DC isolated Converter module
...
+K isolated isolation dc-dc converter step-down dual
+F https://www.xppower.com/portals/0/pdfs/SF_JTD15.pdf
+$ENDCMP
+#
+$CMP JTD1548D15
+D Isolated 15W 4:1 input DC/DC converter module, 18-75V input voltage, +/-15V output voltage, DIP
+K isolated isolation dc-dc converter step-down dual
+F https://www.xppower.com/portals/0/pdfs/SF_JTD15.pdf
+$ENDCMP
+#
```
Comment (chschlue) [inline Converter_DCDC.dcm]: JTD1548S05 missing?

File: Converter_DCDC.lib [hunk 2]
```diff
@@ -505,6 +505,62 @@ X -Vout 8 400 -100 100 L 50 50 0 0 w
...
+P 2 0 1 0 0 0 0 -50 N
+P 2 0 1 0 0 100 0 50 N
+P 2 0 1 0 0 200 0 150 N
+P 2 0 1 0 0 300 0 250 N
+X +VIN 1 -500 200 100 R 50 50 1 1 W
+X -VIN 2 -500 -200 100 R 50 50 1 1 W
+X +VOUT 3 500 200 100 L 50 50 1 1 w
+X Common 4 500 0 100 L 50 50 1 1 w
+X -VOUT 5 500 -200 100 L 50 50 1 1 w
+X On/Off 6 -500 0 100 R 50 50 1 1 I
```
Comment (chschlue) [inline Converter_DCDC.lib]: Please use On/~Off for clarity.

File: Converter_DCDC.lib [hunk 3]
```diff
@@ -505,6 +505,62 @@ X -Vout 8 400 -100 100 L 50 50 0 0 w
...
+P 2 0 1 0 0 0 0 -50 N
+P 2 0 1 0 0 100 0 50 N
+P 2 0 1 0 0 200 0 150 N
+P 2 0 1 0 0 300 0 250 N
+X +VIN 1 -500 200 100 R 50 50 1 1 W
+X -VIN 2 -500 -200 100 R 50 50 1 1 W
+X +VOUT 3 500 200 100 L 50 50 1 1 w
+X Trim 4 500 0 100 L 50 50 1 1 I
+X -VOUT 5 500 -200 100 L 50 50 1 1 w
+X On/Off 6 -500 0 100 R 50 50 1 1 I
```
Comment (chschlue) [inline Converter_DCDC.lib]: Same as above

File: Converter_DCDC.lib [hunk 4]
```diff
@@ -505,6 +505,62 @@ X -Vout 8 400 -100 100 L 50 50 0 0 w
...
+# JTD2024D05
+#
+DEF JTD2024D05 U 0 20 Y Y 1 F N
+F0 "U" -350 350 50 H V C CNN
+F1 "JTD2024D05" 350 350 50 H V C CNN
+F2 "Converter_DCDC:Converter_DCDC_XP_POWER_JTDxxxxxxx_THT" 0 -400 50 H I C CNN
+F3 "" -25 0 50 H I C CNN
+ALIAS JTD2024D12 JTD2024D15 JTD2048D05 JTD2048D12 JTD2048D15 JTD1524D05 JTD1524D12 JTD1524D15 JTD1548D05 JTD1548D12 JTD1548D15
+$FPLIST
+ Converter*DCDC*XP_POWER*JTDxxxxxxx*
```
Comment (chschlue) [inline Converter_DCDC.lib]: Converter\*DCDC\*XP?POWER\*JTDxxxxxxx\*

File: Converter_DCDC.lib [hunk 5]
```diff
@@ -505,6 +505,62 @@ X -Vout 8 400 -100 100 L 50 50 0 0 w
...
+# JTD2024S3V3
+#
+DEF JTD2024S3V3 U 0 20 Y Y 1 F N
+F0 "U" -350 350 50 H V C CNN
+F1 "JTD2024S3V3" 350 350 50 H V C CNN
+F2 "Converter_DCDC:Converter_DCDC_XP_POWER_JTDxxxxxxx_THT" 0 -400 50 H I C CNN
+F3 "" -25 0 50 H I C CNN
+ALIAS JTD2024S05 JTD2024S12 JTD2024S15 JTD2048S3V3 JTD2048S05 JTD2048S12 JTD2048S15 JTD1524S3V3 JTD1524S05 JTD1524S12 JTD1524S15 JTD1548S3V3 JTD1548S12 JTD1548S15
+$FPLIST
+ Converter*DCDC*XP_POWER*JTDxxxxxxx*
```
Comment (chschlue) [inline Converter_DCDC.lib]: Converter\*DCDC\*XP?POWER\*JTDxxxxxxx\*

**Comment by jamesrlucas** [comment]
All done!

**Comment by chschlue** [comment]
Thanks.

**Pull request merged**